Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -229,6 +228,9 @@ scoped[Manifold]
     (modelWithCornersEuclideanHalfSpace n :
       ModelWithCorners ℝ (EuclideanSpace ℝ (Fin n)) (EuclideanHalfSpace n))
 
+lemma modelWithCornersEuclideanHalfSpace_zero {n : ℕ} [NeZero n] :
+    (𝓡∂ n) ⟨fun _ ↦ 0, by norm_num⟩ = 0 := rfl
```

Comment:
Do you need the brackets here?
```suggestion
    𝓡∂ n ⟨fun _ ↦ 0, by norm_num⟩ = 0 := rfl
```

Reply:
I do, otherwise Lean complains `function expected`. (The notation 𝓡∂ takes exactly one argument.)

Reply:
That sounds like a missing `withOverApp` or something like that. Can you start a topic on Zulip?

Reply:
Asked on zulip: let's see what comes out of it.